relayer/submit_transaction.go

```diff
@@ -101,6 +103,7 @@ func ValidateAndSendTransaction(
 		req.Signature,
 	)
 	if err != nil {
+		log.Errorf("failed to call execute: %s", err)
```

Comment:
The message seems a little vague. Maybe something like `failed to simulate forwarder call`? It says this line wasn't unit tested, but we have a unit test, `Test_ValidateAndSendTransaction`, that is supposed to trigger a simulation error. It tries to claim twice to trigger it. I don't have the branch checked out right now, but I could look into it if you don't have time.

Reply:
this isn't the simulation call, this is the actual call to the contract